.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,14 +24,28 @@ Check for answers on StackOverflow with the [google-maps](http://stackoverflow.c
 
 Please be sure to include as much information as possible:
 
+#### Summary
+
+Summarize your issue in one or two sentences
+
+#### Expected behavior
+
+What did you expect?
+
+#### Observed behavior
+
+What did you see instead?  Describe your issue in detail here.
+
 #### Environment details
 
-1. Specify the API at the beginning of the title (for example, "Places: ...")
-2. OS type and version
-3. Library version and other environment information
+1. Device make and model, or emulator details
+1. Android version
+1. Library version and other environment information
```

Comment:
Renumber as 1, 2, & 3?

Reply:
When this is rendered GitHub will convert these to actual numbers - it's the lazy persons way of numbering lists so you don't have to renumber then when items change 😉:

~~~
1. First
1. Second
1. Third
~~~

1. First
1. Second
1. Third

Reply:
Although the developer will see these unrendered 🤦‍♂. I'll change it now...

Reply:
Renumbered in https://github.com/googlemaps/android-maps-utils/pull/667/commits/d97174181d4db7f8f7673c51da3ff06a010c95e9